features/layout/footer/footer.tsx

```diff
@@ -0,0 +1,35 @@
+import packageJson from "package.json";
```

Comment:
Importing package.json into your client-side code is considered potentially dangerous. There are some alternatives like creating an API route that exposes the version number via an API call.

My favorite solution is to use the [next.js config](https://nextjs.org/docs/pages/api-reference/next-config-js/env). You can import package.json safely there and assign the version number to an environment variable. Then you can read this env variable in the client side code. This solution has the smallest overhead imo